README.md

```diff
@@ -70,7 +70,7 @@ This will prompt you to make a "Pull Request" in stdout, click on it, make PR.
       * ~~Dictionaries~~
       * ~~Modules and Packages~~
   * Advanced Tutorials
```

Comment:
Do you mind also resetting the order of these to the original tutorial order (and add in the missing ones)

My order is actually pretty scuffed lol